ci-operator/config/stolostron/search-v2-operator-bundle/stolostron-search-v2-operator-bundle-main.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  stolostron_builder_go1.18-linux:
+    name: builder
+    namespace: stolostron
+    tag: go1.18-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: stolostron
+    tag: go1.18-linux
+images:
+- dockerfile_path: bundle.Dockerfile
+  from: base
+  to: search-operator-bundle
+promotion:
+  name: "2.7"
+  namespace: stolostron
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: pr-bundle-image-mirror
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: search-operator-bundle
+    env:
+      IMAGE_REPO: search-operator-bundle
+      REGISTRY_ORG: stolostron
+    workflow: ocm-ci-image-mirror
+- as: git-fast-forward
+  postsubmit: true
+  steps:
+    env:
+      DESTINATION_BRANCH: release-2.7
+    workflow: ocm-ci-fastforward
+zz_generated_metadata:
+  branch: main
+  org: stolostron
+  repo: search-v2-operator-bundle
```

Comment:
@dislbenn - I see some 2.5 and 2.6 failures , we care only about 2.7 and main configurations - We can leave the 2.5 and 2.6 prow jobs as is . If that helps .